charts/krakend-operator/templates/replicationconf.yaml

```diff
@@ -1,4 +1,5 @@
 ---
+{{- if .Values.enableReplicationConf }}
```

Comment:
This conditions’ variable naming is awful.
It requires strong deduction abilities to determine what exactly it enables.

As it was discussed it is rather better to complete removal of this weird manifest template out of KRAKEND-OPERATOR helm chart as it is far beyond of its scope.